### PR TITLE
Enhance documentation for KubeVirt VirtualMachineInstancePreset

### PR DESCRIPTION
**Pull request by hdurand0710**
Signed-off-by: Helene Durand <helene@kubermatic.com>

### CODE REVIEW

File: content/kubermatic/master/architecture/supported-providers/kubevirt/kubevirt.en.md [hunk 1]
```diff
@@ -130,11 +149,60 @@ virtualMachine:
...
+- `CPU`
+- `Firmware`
+- `Clock`
+- `Features`
+- `Devices.Watchdog`
+- `IOThreadsPolicy`
+
+*Note3:* Migration to the instanceType API
+
+Please note that in the next KKP release, we will migrate to the new instanceType API, as the VirtualMachineInstance will be deprecated.
```
Comment (mfranczy) [inline content/kubermatic/master/architecture/supported-providers/kubevirt/kubevirt.en.md:203]: please change to:
> as the VirtualMachineInstancePreset

somebody might think that VirtualMachineInstance object  will be deprecated :) We have to be very specific about this.

Comment (mfranczy) [inline content/kubermatic/master/architecture/supported-providers/kubevirt/kubevirt.en.md:203]: I see that instance type API is already released as [alpha](https://github.com/kubevirt/api/blob/main/instancetype/v1alpha1/types.go#L35), so we can change `instanceType` to `VirtualMachineInstanceType`

File: content/kubermatic/master/architecture/supported-providers/kubevirt/kubevirt.en.md [hunk 2]
```diff
@@ -130,11 +149,60 @@ virtualMachine:
...
+This will for example lift the limitation around the list of merged fields from the `VirtualMachineInstance`.
```
Comment (mfranczy) [inline content/kubermatic/master/architecture/supported-providers/kubevirt/kubevirt.en.md:204]: additionaly I would add  
> and provide deterministic behavior with immediate detection of merge conflicts.

**Review by mfranczy — CHANGES_REQUESTED** [review]
Overal looks good, a few small comments added.

File: content/kubermatic/master/architecture/supported-providers/kubevirt/kubevirt.en.md (hunk 1 above)
Comment (hdurand0710) [reply content/kubermatic/master/architecture/supported-providers/kubevirt/kubevirt.en.md:203]: very true.

Comment (hdurand0710) [reply content/kubermatic/master/architecture/supported-providers/kubevirt/kubevirt.en.md:203]: Should I link to the alpha API in the doc ? I would say yes, but what do you think ?

File: content/kubermatic/master/architecture/supported-providers/kubevirt/kubevirt.en.md (hunk 2 above)
Comment (hdurand0710) [reply content/kubermatic/master/architecture/supported-providers/kubevirt/kubevirt.en.md]: done.

File: content/kubermatic/master/architecture/supported-providers/kubevirt/kubevirt.en.md (hunk 1 above)
Comment (mfranczy) [reply content/kubermatic/master/architecture/supported-providers/kubevirt/kubevirt.en.md]: Doesn't harm, it's ok.

**Review by mfranczy — APPROVED** [review]
/approve
/lgtm

**Pull request merged**